backend/ledgers.ts

```diff
@@ -12,94 +10,124 @@ interface Ledger {
   operation_count: number;
 }
 
-interface LedgerSql {
-  date: string;
-  transaction_count: string;
-  operation_count: string;
-}
+// TODO - use any until https://github.com/stellar/js-stellar-sdk/issues/731 resolved
+export type LedgerRecord = any;
+
+const REDIS_LEDGER_KEY = "ledgers";
+const REDIS_PAGING_TOKEN_KEY = "paging_token";
 
 export const handler = async function (
   _: any,
   res: Response,
   next: NextFunction,
 ) {
   try {
-    const cachedData = await getOrThrow(redisClient, "ledgers");
+    const cachedData = await getOrThrow(redisClient, REDIS_LEDGER_KEY);
     const ledgers: Array<Ledger> = JSON.parse(cachedData as string);
     res.json(ledgers);
   } catch (e) {
     return next(e);
   }
 };
 
-function updateResults() {
-  let query = `select
-      to_char(closed_at, 'YYYY-MM-DD') as date,
-      sum(transaction_count) as transaction_count,
-      sum(operation_count) as operation_count
-    from ledger_stats
-    where closed_at >= current_date - interval '29' day
-    group by to_char(closed_at, 'YYYY-MM-DD')
-    order by 1 desc`;
-
-  postgres.sequelize
-    .query(query, { type: QueryTypes.SELECT })
-    .then((results: Array<LedgerSql>) => {
-      const cachedData: Array<Ledger> = map(results, convertFields);
-      redisClient.set("ledgers", JSON.stringify(cachedData));
+export async function updateLedgers() {
+  let pagingToken = await redisClient.get("paging_token");
+  if (pagingToken == null || pagingToken == "") {
+    pagingToken = "now";
```

Comment:
We could make "now" string a const as well to stay consistent with other string consts.

Reply:
https://github.com/stellar/dashboard/pull/209/commits/3aa116a54534ef1f93a1d32d26964f38c83c04dd